core/src/test/scala/com/pingcap/tispark/datasource/RowIDAllocatorSuite.scala

```diff
@@ -0,0 +1,42 @@
+package com.pingcap.tispark.datasource
+
+import com.pingcap.tikv.allocator.RowIDAllocator
+import org.apache.spark.sql.BaseTiSparkSuite
+
+class RowIDAllocatorSuite extends BaseTiSparkSuite {
+  test("test unsigned allocator") {
+    tidbStmt.execute("drop table if exists t")
```

Comment:
will conflict on table `t` if enable parallel test?

Reply:
it might. I will use a more specific name.

Reply:
agree